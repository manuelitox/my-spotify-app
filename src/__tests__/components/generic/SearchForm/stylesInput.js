import React from 'react'
import { shallow } from 'enzyme'

import StyledInput from 'components/generic/SearchForm/stylesInput'

describe('Component | StyledInput', () => {
  let component 
  beforeEach( () => {
    component = shallow(
      <StyledInput />
    )
  })
  it('should render a styled component', () => {
    expect(component).toMatchSnapshot()
  })
})