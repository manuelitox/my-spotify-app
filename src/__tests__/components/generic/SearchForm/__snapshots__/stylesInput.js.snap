// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component | StyledInput should render a styled component 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
  appearance: none;
  border: 0;
  border-radius: ",
          [Function],
          ";
  padding: 20px;
  &:focus {
    outline: 0;
  }
",
        ],
      },
      "displayName": "styled.input",
      "foldedComponentIds": Array [],
      "propTypes": Object {
        "isOpen": [Function],
      },
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "input",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
